rootfs/etc/cont-init.d/01-acarsdec

```diff
@@ -34,8 +34,7 @@ for i in "${SPLIT_FREQS[@]}"
 do
 :
 if [[ $(echo "$i > 118.0" | bc)  -eq 0 || $(echo "$i < 137.0" | bc) -eq 0 ]]; then
-	echo "FREQUENCY $i is not in the range of 118.0 - 137.0, exiting"
-	exit 1
+	echo "FREQUENCY $i is not in the range of 118.0 - 137.0"
```

Comment:
I'm debating whether it matters one way or the other, but it doesn't exit on the freq range check if it encounters an error when it did before. I verified what acarsdec does and it looks like it takes the frequency out of range but spits out a warning, so this does follow the behavior of acarsdec.